e2e/js/js-cloud-server/tests/js-cloud-server.spec.ts

```diff
@@ -0,0 +1,7 @@
+describe('js-cloud-server', () => {
+    it('should return all variables', async () => {
+        const response = await fetch('http://localhost:8787')
+        const text = await response.text()
+        expect(text).toContain('DevCycle Variables')
```

Comment:
should test the actual output from a project using the `.variable` function, use this project https://app.devcycle.com/o/org_U9F8YMaTChTEndWw

Reply:
fixed it